src/Perlang.Interpreter/Compiler/PerlangCompiler.cs

```diff
@@ -1542,7 +1542,7 @@ public VoidObject VisitVarStmt(Stmt.Var stmt)
 
         currentMethod.Append($"{Indent(indentationLevel)}{stmt.TypeReference.PossiblyWrappedCppType} {variableName}");
 
-        if (stmt.HasInitializer)
+        if (stmt.Initializer != null)
```

Comment:
This kind of change :point_up: is actually more than just "removing a property getter". It's actually _required_ now that `Initializer` is nullable, to avoid CS8062 nullability warnings.